crafts.lua

```diff
@@ -129,6 +129,12 @@ local tool_materials = {
 	["mithril"] = {"moreores:mithril_ingot", 1, "moreores"},
 }
 
+-- assume mese unmeltable unless mesecons_wires exists
+if core.get_modpath("mesecons_wires") then
```

Comment:
`crafts.lua:133:4: accessing undefined variable core`
```suggestion
if minetest.get_modpath("mesecons_wires") then
```

Reply:
I guess changing the whole namespace thing belongs in another pr